airflow/hooks/druid_hook.py

```diff
@@ -14,9 +14,12 @@
 
 from __future__ import print_function
 
+from contextlib import closing
 import requests
+import sys
 import time
 
+from pydruid.db import connect
```

Comment:
nit: space bellow to group out by standard lib, external lib, and inside-package imports

Reply:
thanks for the suggestion. will update the pr  :)